ledger/src/blockstore.rs

```diff
@@ -7154,10 +7113,26 @@ pub mod tests {
     #[test]
     fn test_should_insert_data_shred() {
```

Comment:
Can you help annotate this one for me? I'm not following why we had to switch from `make_slot_entries` to `entries_to_shreds` and some of the magic number changes

Reply:
We need to make all shreds in this test come from the same source (same keypair). The magic number is the number of ticks to produce, I've set it to a higher amount to make sure that the amount of stuff in shreds is > 1 FEC set long, such that the slot termination shred is not coming too soon.

Reply:
> I've set it to a higher amount to make sure that the amount of stuff in shreds is > 1 FEC set long

Can you add something like this  immediately after shred creation to have some self-documenting code:
```rust
assert(shreds.len() > DATA_SHREDS_PER_FEC_BLOCK);
```